assets/css/style.css

```diff
@@ -437,14 +440,20 @@ input {
     flex-direction: row-reverse;
   }
 
-  .article-btn:active {
+  .article-btn:hover {
     background-color: #396df2;
     color: #fff;
-    box-shadow: 1px 4px 4px #aaa;
+    box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);
+  }
+
+  .article-btn:active {
+    background-color: #5280f5;
+    color: #fff;
+    box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);
   }
 
   .article-item:hover {
-    box-shadow: 1px 10px 10px #ccc;
+    box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);
```

Comment:
- [ ] Good job making almost all the required changes @WilfriedM01, But as mentioned earlier, The changes are reflected only on the desktop design, We need to have the same effects on the mobile design. Please notice the recording I have attached to you down. 
> Please consider applying the changes outside of the media query. that way your mobile design will also have the effects. 

https://github.com/WilfriedM01/portfolio/assets/69077061/e4b72f8b-aec8-47c6-865b-e9a6a56b5741